### PR TITLE
Fix metastore bindings and add sample configurations

### PR DESCRIPTION
**Pull request by 7c00**
This PR  
1.  fixes a minor bug on dependency bingdings of metastore related classes;
2.  adds sample configurations for hudi connector;
3. bundles hudi connector to presto-server;

Test plan - No tests needed.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by codope — APPROVED** [review]
LGTM.
@arunthirupathi @highker Could you please review this?

File: presto-server/pom.xml [hunk 1]
```diff
@@ -364,6 +364,14 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```
Comment (arunthirupathi) [inline presto-server/pom.xml:367]: What is the size before and after this change

Comment (7c00) [reply presto-server/pom.xml:367]: The total size of `presto-server` is 1856MB (uncompressed, or 1681 compressed), including 193MB of `plugin/hudi`.

Comment (arunthirupathi) [reply presto-server/pom.xml:367]: Can you please do a build before this change and after this change and report the number ?

Comment (7c00) [reply presto-server/pom.xml:367]: The size changes are

. | presto-server-0.274-SNAPSHOT/ | presto-server-0.274-SNAPSHOT.tar.gz 
---- | ---- | ----
Before | 1663MB   | 1505MB
After | 1856MB  | 1681MB
+/- | +193MB   | +176MB

**Review by arunthirupathi — APPROVED** [review]

**Comment by 7c00** [comment]
Thanks @arunthirupathi @codope

**Pull request merged**